core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -1508,6 +1508,54 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     servers.foreach(assertNoExemptRequestMetric(_))
   }
 
+  @Test
+  def testRateMetricsHaveCumulativeCount() {
```

Comment:
Maybe worth adding a method comment saying that we test both the producer and consumer here.